jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/SsoSessionConfigurationManager.kt

```diff
@@ -57,7 +58,7 @@ class SsoSessionConfigurationManager {
 
         if (ssoSessionSection?.isEmpty == false) {
             val existing = """
-            [$SSO_SESSION_SECTION_NAME $ssoProfileName]
+            [$SSO_SESSION_SECTION_NAME $idcProfileName]
```

Comment:
test?

Reply:
We'll include this test case, along with the others we talked about during our stand-up meeting. This PR includes only, the tests specifically for the IDC profile and SSO session